config/locales/ja.yml

```diff
@@ -226,6 +231,7 @@ ja:
       content: セキュリティ認証に失敗しました。Cookieをブロックしていませんか？
       title: セキュリティ認証に失敗
     '429': リクエストの制限に達しました。
+    noscript: Mastodonを利用する場合はJavaScriptを有効にしてください。
```

Comment:
Certainly "use" has a slight incongruity. However, I do not want to change it significantly from the original phrase.

The difference between "使用" and "利用" is difficult. However, since it is provided as a web service, I thought that "利用" is more appropriate.